index.js

```diff
@@ -65,33 +65,25 @@ function oust(src, type, raw = false) {
     }
   }
 
-  const chosenTypes = typeArray.map(type => ({...types[type], type}));
-  const selector = chosenTypes.map(type => type.selector).join(', ');
   const $ = cheerio.load(src);
+  const chosenTypes = typeArray.map(type => ({...types[type], type}));
+  const $selector = $(chosenTypes.map(type => type.selector).join(', '));
 
-  return Array.prototype.map.call($(selector), element => {
-    const $element = $(element);
-    const chosenType = chosenTypes.find(type => $element.is(type.selector));
-
+  return [...$selector].map(element => {
+    const $el = $(element);
+    const {type, method, attribute} = chosenTypes.find(type => $el.is(type.selector));
```

Comment:
this breaks if `chosenTypes.find` returns `undefined`
```suggestion
    const {type, method, attribute} = chosenTypes.find(type => $el.is(type.selector)) || {};
```

Reply:
Thanks! I had the same feeling too, but I wasn't sure if it does indeed return undefined since we don't seem to have a test for it.